clean: Use standard cache directories in linux and darwin (#241)
codacy/codacy-coverage-reporter: docs/alternative-ways-of-running-coverage-reporter.md

wget -qO - https://coverage.codacy.com/get.sh | sh
```

> The self-contained script is able to cache the binary. To avoid having to download it everytime,
> you need to add one of the following directories to your CI cached folders:
> 
> - `$HOME/.cache/codacy` on Linux
> - `$HOME/Library/Caches/Codacy` on Mac OS X

Reviewer: Small language edits and make the list item indentation consistent:

```suggestion
The self-contained script is able to cache the binary. To avoid downloading the binary every time that the script runs, add one of the following directories to your CI cached folders:

-   `$HOME/.cache/codacy` on Linux
-   `$HOME/Library/Caches/Codacy` on Mac OS X
```
